### PR TITLE
Add doc-entry for rebroadcast endpoint [ECR-2131]

### PR DESCRIPTION
**Pull request by VladYermakov**
Added doc entry for `v1/rebroadcast` endpoint.

### CODE REVIEW

**Comment by DarkEld3r** [comment]
@slowli Done: this pull request now targets `develop` branch.

File: src/advanced/node-management.md [hunk 1]
```diff
@@ -429,6 +429,30 @@ curl -H "Content-Type: application/json" --data 'null' http://127.0.0.1:7780/api
 null
 ```
 
+### Rebroadcast
+
+```none
+POST {system_base_path}/rebroadcast
+```
+
+Rebroadcast all transactions from the pool to other validators.
```
Comment (dmitry-timofeev) [inline src/advanced/node-management.md]: Why would a user need this endpoint (I'd explain since this is a reference documentation)?

But really, why is that needed? Can't Exonum exchange transactions properly?

Comment (DarkEld3r) [reply src/advanced/node-management.md]: > Can't Exonum exchange transactions properly?

No, it cannot in some very specific cases. For example, we have a network with some validators which are all off-line at the moment and one auditor. A user sends a transaction to the auditor, who broadcasts it to the validators (but they are off-line). As the auditors (a validator can become a leader and propose a block with such transaction) don't have transactions exchange mechanic, this transaction will stuck forever in the pool.

> Why would a user need this endpoint (I'd explain since this is a reference documentation)?

Generally, a user doesn't. This is a temporary workaround, so I'm not sure it should be explained thoroughly.

Comment (dmitry-timofeev) [reply src/advanced/node-management.md]: I see — but not sure that I understand completely how an auditor can become a leader and propose transactions?
